iot-hub/Samples/service/AutomaticDeviceManagementSample/AutomaticDeviceManagementSample.cs

```diff
@@ -90,24 +99,40 @@ private static void CreateMetricsAndTargetCondition(Configuration configuration)
 
         private async Task DeleteConfigurationAsync(string configurationId)
         {
-            await _registryManager.RemoveConfigurationAsync(configurationId);
+            try
+            {
+                await _registryManager.RemoveConfigurationAsync(configurationId);
+            }
+            catch (ThrottlingException)
+            {
+                Console.WriteLine("Too many jobs scheduled at this given time. Please try again later.");
+                return;
+            }
 
             Console.WriteLine($"Configuration deleted, id: {configurationId}");
         }
 
         private async Task GetConfigurationsAsync(int count)
         {
-            IEnumerable<Configuration> configurations = await _registryManager.GetConfigurationsAsync(count);
+            try
+            {
+                IEnumerable<Configuration> configurations = await _registryManager.GetConfigurationsAsync(count);
+
+                // Check configuration's metrics for expected conditions
+                foreach (Configuration configuration in configurations)
+                {
+                    string configurationString = JsonConvert.SerializeObject(configuration, Formatting.Indented);
+                    Console.WriteLine(configurationString);
+                    Thread.Sleep(1000);
+                }
 
-            // Check configuration's metrics for expected conditions
-            foreach (Configuration configuration in configurations)
+                Console.WriteLine("Configurations received");
+            }
+            catch (ThrottlingException)

```

Comment:
There doesn't appear to be a service request in the try block, so no ThrottlingException will ever be thrown. Because of that, you can remove the try/catch here